website/source/docs/configuration/interpolation.html.md

```diff
@@ -312,6 +312,12 @@ The supported built-in functions are:
   * `pathexpand(string)` - Returns a filepath string with `~` expanded to the home directory. Note:
     This will create a plan diff between two different hosts, unless the filepaths are the same.
 
+  * `pow(x, y)` - Returns the base `x` of exponential `y`.
```

Comment:
The `math.Pow()` function requires two `float64` numbers. And inside the interpolation function you're casting the supplied interfaces to `float64`. This will also work on integers, but we should document that the function takes in floats.